aspnetcore/tutorials/choose-web-ui.md

```diff
@@ -3,83 +3,56 @@ title: Choose an ASP.NET Core UI
-
-### Client rendered UI
-
-A client rendered app dynamically renders web UI on the client, directly updating the browser DOM as necessary.
-
-Benefits:
-
-* Allows for rich interactivity that is nearly instant, without requiring a round trip to the server. UI event handling and logic run locally on the user's device with minimal latency. 
-* Supports incremental updates, saving partially completed forms or documents without the user having to select a button to submit a form.
-* Can be designed to run in a disconnected mode. Updates to the client-side model are eventually synchronized back to the server once a connection is re-established.
-* Reduced server load and cost, the work is offloaded to the client. Many client rendered apps can also be hosted as static websites.
-* Takes advantage of the capabilities of the user’s device.
+* Create rich interactive UIs using [C#](/dotnet/csharp/) instead of [JavaScript](https://www.javascript.com).
+* Render the UI as HTML and CSS for wide browser support, including mobile browsers.
+* Integrate with modern hosting platforms, such as [Docker](/dotnet/standard/microservices-architecture/container-docker-introduction/index).
+* Build hybrid desktop and mobile apps with .NET and Blazor.
 
-Examples of client rendered web UI:
+Blazor Web Apps support interactivity with client rendering that relies on a .NET runtime built with [WebAssembly](https://webassembly.org) that you can download with your app. When running Blazor on WebAssembly, your .NET code can access the full functionality of the browser and interop with JavaScript. Your .NET code runs in the browser's security sandbox with the protections that the sandbox provides against malicious actions on the client machine.
 
-* An interactive dashboard.
-* An app featuring drag-and-drop functionality
-* A responsive and collaborative social app.
+Running .NET code inside web browsers is made possible by [WebAssembly](https://webassembly.org) (abbreviated `wasm`). WebAssembly is a compact bytecode format optimized for fast download and maximum execution speed. WebAssembly is an open web standard and supported in web browsers without plugins. Blazor WebAssembly works in all modern web browsers, including mobile browsers.
 
-Drawbacks:
+When a Blazor WebAssembly is built and run:
 
-* Code for the logic has to be downloaded and executed on the client, adding to the initial load time.
-* Client requirements may exclude users who have low-end devices, older browser versions, or low-bandwidth connections.
+* C# code files and Razor files are compiled into .NET assemblies.
+* The assemblies and the [.NET runtime](/dotnet/framework/get-started/overview) are downloaded to the browser.
+* Blazor WebAssembly bootstraps the .NET runtime and configures the runtime to load the assemblies for the app. The Blazor WebAssembly runtime uses JavaScript interop to handle DOM manipulation and browser API calls.
```

Comment:
This is a lot of content on how Blazor can run on WebAssembly. I don't think we really need any of this in this doc.

Reply:
Removed content on how Blazor can ron on WebAssembly, thanks.